Task1/readFile.js

```diff
@@ -0,0 +1,54 @@
+// READ FILE RECURSIELY & Create Hash. Also store result to Output file
+
+const fs = require('fs');
+const path = require('path');
+const crypto = require('crypto');
+const outputFile = "result.txt";
+const myArgs = process.argv.slice(2);
+const myPath = myArgs[0];
+
+fs.writeFileSync(outputFile, "--- Generating Fresh Output file ----\n");
+
+// You can pass your algorithm here like md5,sha1
+function fileToHash(file,algo)
+{
+	let file_buffer = fs.readFileSync(file);
+	let sum = crypto.createHash(algo);
+	sum.update(file_buffer);
+	return sum.digest('hex');
+}
+
+// Result is stored in outputFile 
+function writeToFile(data){
+	fs.appendFileSync(outputFile, data);
+}
+
+// Reads files recursively and create hash
+function readFiles(path){	
+	var files = fs.readdirSync(path);
+		for(file of files){
+			let fileFullPath = __dirname+"\\"+path+"\\"+file;
```

Comment:
here you can make use of `path` module to handle path concatenation supporting all Operating Systems.